global_helpers/panther_aws_ami.yml

```diff
@@ -0,0 +1,5 @@
+AnalysisType: global
+Filename: panther_aws_ami.py
```

Comment:
Do we need a new helper? Could we use an existing one? If not, can we rename this to be more specific that it's about vuln-xz-aws-amis?

Reply:
`panther_iocs` may be a good fit for it, actually. I'll refactor this.

Reply:
Changed in [`c78dd0c` (#1182)](https://github.com/panther-labs/panther-analysis/pull/1182/commits/c78dd0c744c5121ace4781433931269389d36cdf).